From: =?utf-8?q?Lisandro_Dami=C3=A1n_Nicanor_P=C3=A9rez_Meyer?=
 <lisandro@debian.org>
Date: Mon, 20 Nov 2023 01:57:46 +0000
Subject: remove non-used privacy-breach code

Forwarded: not-needed
Last-Update: 2015-02-18

This code makes Lintian unhappy. But we are really not using it, it only
gets inserted when building the online doc.
Anyways the best way to calm down Lintian is to simply remove it.
---
 doc/global/template/scripts/main.js | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/doc/global/template/scripts/main.js b/doc/global/template/scripts/main.js
index 823cebe..7d99e00 100644
--- a/doc/global/template/scripts/main.js
+++ b/doc/global/template/scripts/main.js
@@ -94,11 +94,6 @@ $(document).ready(function($) {
 });
 
 $( window ).load(function() {
-    load_sdk('script', 'facebook-jssdk','//connect.facebook.net/en_US/sdk.js#xfbml=1&appId=207346529386114&version=v2.0');
-    load_sdk('script', 'twitter-wjs', '//platform.twitter.com/widgets.js');
-    $.getScript("//www.google.com/jsapi", function(){
-        google.load("feeds", "1", {"callback": oneQt.liveFeeds});
-    });
 });
 
 var oneQt = {
